concepts/05 UI Components/Popup/10 Show and Hide the Popup/15 Using DxPopupService.md

```diff
@@ -0,0 +1,77 @@
+To use [services](https://angular.dev/guide/di/creating-injectable-service) for popup opening is a common practice in Angular libraries. DevExtreme Popup allows you to use `DxPopupService` to initiate popups directly from .ts files without involving a template code.
+
+`DxPopupService` is imported from `'devextreme-angular/ui/popup'` and functions as a typical Angular service. The service has one method, `open`, which takes two arguments:
+
+1. The Angular component class to be used as the content of the popup.
+2. The popup configuration, which includes [properties](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/) of the Popup component.
+
+The `open` method returns an object of the `DxPopupServiceComponent` type. This type is similar to `DxPopupComponent` but includes a `contentRef` property. Use `contentRef.instance` to access the instance of the component set as the popup content.
```

Comment:
I'm just shortening a few phrases. 

"similar to X, but includes Y" -> "extends X with Y" (I hope we can say that)

"to access the instance of the component set as ..." -> "to access the component that serves as"

```suggestion
The `open` method returns a `DxPopupServiceComponent` object. This type extends `DxPopupComponent` with a `contentRef` property. Use `contentRef.instance` to access the component that serves as popup content.
```